tests/resources/hpo_complex_your_own_data.json

```diff
@@ -0,0 +1,143 @@
+{
+  "metadata": {
```

Comment:
this file doesn't appear to be used anywhere

Reply:
Removed in 11dc4cac77b11b822ac1c1e160a5ef5c03b7bca7.